src/main/java/pl/edu/zut/mad/schedule/model/inner/Schedule.java

```diff
@@ -217,7 +233,9 @@ public String getSubstituteName() {
         RESERVATION_STATUS_ABBREVIATION("reservationStatusAbbreviation", Schedule::getReservationStatusAbbreviation),
         STATUS("status", Schedule::getStatus),
         SUBSTITUTE_SURNAME("substituteSurname", Schedule::getSubstituteSurname),
-        SUBSTITUTE_NAME("substituteName", Schedule::getSubstituteName);
+        SUBSTITUTE_NAME("substituteName", Schedule::getSubstituteName),
+        GROUP("group", Schedule::getGroup),
```

Comment:
It's not clear for me what really means simple `GROUP` when we have also `GROUP_NAME`